lighthouse/datadog_checks/lighthouse/lighthouse.py

```diff
@@ -0,0 +1,83 @@
+# the following try/except block will make the custom check compatible with any Agent version
+try:
+    # first, try to import the base class from old versions of the Agent...
+    from checks import AgentCheck
+except ImportError:
+    # ...if the above failed, the check is running in Agent version 6 or later
+    from datadog_checks.checks import AgentCheck
+    from datadog_checks.errors import CheckException
+
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+import json
+
+# content of the special variable __version__ will be shown in the Agent status page
+__version__ = "1.0.0"
```

Comment:
You can remove those lines, the `__version__` variable in your `__about__.py` file is the one being used for an integration.